src/utils/index.js

```diff
@@ -130,6 +133,10 @@ var isIOS = module.exports.isIOS = function () {
   return /iPad|iPhone|iPod/.test(navigator.platform);
 };
 
+var isGearVR = module.exports.isGearVR = function () {
+  return /SamsungBrowser.+Mobile VR/i.test(navigator.userAgent);
```

Comment:
nice, thanks! I'd suggest also making a PR against https://github.com/borismus/webvr-polyfill/blob/99c2649/src/util.js#L184 and https://github.com/borismus/webvr-boilerplate/blob/ecd6afc/src/util.js#L22